lib/elixir/lib/system.ex

```diff
@@ -5,6 +5,30 @@ defmodule System do
   with the VM or the host system.
   """
 
+  @typedoc """
+  The time unit that can be passed to functions like `System.system_time/1`
+  and similar.
+
+  The `:seconds`, `:milli_seconds`, `:micro_seconds`, and `:nano_seconds` time
+  units are self-explanatory: the return value of the functions that accept a
+  time unit will be in the given time unit.
+
+  A time unit can also be a strictly positive integer. In this case, it
+  represents the "parts per second": the time will be returned in
+  `1 / parts_per_second` seconds. For example, the `:milli_seconds` time unit
+  can be simulated by using `1000` as the time unit (as the time will be
```

Comment:
Good point, updated with new phrasing (fixed the unbalanced parens in the other comment as well).